client/netmap.go

```diff
@@ -196,3 +196,83 @@ func (c *Client) NetworkInfo(ctx context.Context, prm PrmNetworkInfo) (*ResNetwo
 
 	return &res, nil
 }
+
+// PrmNetMapSnapshot groups parameters of NetMapSnapshot operation.
+type PrmNetMapSnapshot struct {
+	prmCommonMeta
+}
+
+// ResNetMapSnapshot groups resulting values of NetMapSnapshot operation.
+type ResNetMapSnapshot struct {
+	statusRes
+
+	netMap netmap.NetMap
+}
+
+// NetMap returns current server's local network map.
+func (x ResNetMapSnapshot) NetMap() netmap.NetMap {
+	return x.netMap
+}
+
+// NetMapSnapshot requests current network view of the remote server.
+//
+// Any client's internal or transport errors are returned as `error`.
+// If PrmInit.ResolveNeoFSFailures has been called, unsuccessful
+// NeoFS status codes are returned as `error`, otherwise, are included
+// in the returned result structure.
+//
+// Context is required and MUST NOT be nil. It is used for network communication.
+//
+// Exactly one return value is non-nil. Server status return is returned in ResNetMapSnapshot.
+// Reflects all internal errors in second return value (transport problems, response processing, etc.).
+//
+// Return statuses:
+//   - global (see Client docs).
+func (c *Client) NetMapSnapshot(ctx context.Context, prm PrmNetMapSnapshot) (*ResNetMapSnapshot, error) {
+	// check context
+	if ctx == nil {
+		panic(panicMsgMissingContext)
+	}
+
+	// form request
+	var req v2netmap.SnapshotRequest
+
+	// init call context
+
+	var (
+		cc  contextCall
+		res ResNetMapSnapshot
+	)
+
+	c.initCallContext(&cc)
+	cc.meta = prm.prmCommonMeta
+	cc.req = &req
+	cc.statusRes = &res
+	cc.call = func() (responseV2, error) {
+		return rpcapi.NetMapSnapshot(&c.c, &req, client.WithContext(ctx))
+	}
+	cc.result = func(r responseV2) {
```

Comment:
We have started to refactor client, with the goal of being able to follow all method code on one screen/file.
What about implementing `NetmapSnapshot` in a similar fashion? E.g. see https://github.com/nspcc-dev/neofs-sdk-go/pull/323/commits/70b6c4590ed2dafc8c1fdeb5027a2ad958a49739